cmd/nitro/nitro.go

```diff
@@ -464,6 +464,7 @@ func mainImpl() int {
 	}
 
 	fatalErrChan := make(chan error, 10)
+	quitNodeChan := make(chan string, 10)
```

Comment:
Not sure why we use 10 here, how about adding a comment related to this constant?

Reply:
yeah actually i just copied it from fatalErrChan, but we dont need 10 for quitNodeChan since we are only using it in one place, changed now.